Plan/api/src/main/java/com/djrapitops/plan/extension/builder/ValueBuilder.java

```diff
@@ -228,14 +228,14 @@ default ValueBuilder formatAsTimeAmount() {
      * <p>
      * {@link ValueBuilder#buildBoolean(boolean)}
      */
-    DataValue<Boolean> buildBoolean(Supplier<Boolean> value);
```

Comment:
API classes can not get signature changes like this - it breaks the API, so this needs to be reverted.